lib/facility.rb

```diff
@@ -1,14 +1,74 @@
+require './lib/registrant'
+
 class Facility
-  attr_reader :name, :address, :phone, :services
+  attr_reader :name, 
+              :address, 
+              :phone, 
+              :services, 
+              :collected_fees, 
+              :registered_vehicles
 
-  def initialize(name, address, phone)
+  def initialize(name:, address:, phone:)
     @name = name
     @address = address
     @phone = phone
     @services = []
+    @collected_fees = 0
+    @registered_vehicles = []
+  end
+
+  def add_service(service)
+    @services.push(service)
+  end
+
+  def register_vehicle(vehicle)
+    if @services.include?('Vehicle Registration')
+      @registered_vehicles.push(vehicle)
+      vehicle.registration_today(Date.today)
+      if vehicle.electric_vehicle? == true
+        vehicle.plate_version(:ev)
+        @collected_fees += 200
+        
+      elsif vehicle.antique? == true
+        vehicle.plate_version(:antique)
+        @collected_fees += 25
+      else #regular
+        vehicle.plate_version(:regular)
+        @collected_fees += 100
+      end
+    else
+      nil
```

Comment:
Yup, good job with your flow control here. In general, with an if statement, if you're doing a comparison to a boolean value, you usually don't need to. For example, line 28, if you just had `if vehicle.electric_vehicle?` that would work fine because that statement already gives you back a true/false value.